content/how-to/add-user-ad-saml/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Adding a BluBracket user with Azure AD SAML"
+description: "How to add a BluBracket user with Azure AD SAML"
+lead: "How to integrate with Azure AD SAML"
+date: 2022-01-12T02:48:57+00:00
+draft: false
+weight: 300
+toc: true
+resources:
+- src: azureactivedirectory.png
+- src: enterpriseapplications.png
+---
+
+This article covers the SAML 2.0 authentication requests and responses that BluBracket supports for Single Sign-On (SSO) using Azure Active Directory (Azure AD).
+
+BluBracket requires an email attribute in order to successful identify and authenticate users; BluBracket looks for this vaule from the SAML identity claim http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress.
+
+As of the last update to this article users must exists in BluBracket before they are able to login successfully using Single Sign-On.  BluBracket does not automatically register new users that do not previously exist in the tenant.
+
+1. Login to Azure AD as an Administrator [https://portal.azure.com](https://portal.azure.com)
```

Comment:
Is this really a single 19 step task, or is it a set of smaller tasks? It looks like:

1. Register a new Azure Active Directory app
2. Grant permissions to users in Azure AD to use BluBracket
3. Specify the Azure AD connection details in BluBracket
4. Add users in the BluBracket portal